plugins/modules/site_workflow_manager.py

```diff
@@ -117,20 +121,39 @@
                 description: Floor number within the building site (e.g., 5). This value can only be specified during the creation of the
                     floor and cannot be modified afterward.
                 type: int
-
+              bulk_operation:
+                description: Specifies the type of site (e.g., "bulk"). This field categorizes the site based on its creation method.
+                type: str
 requirements:
-- dnacentersdk == 2.4.5
+- dnacentersdk == 2.9.2
```

Comment:
it should be supported in older versions also..

Reply:
Changed to "2.4.5"